acceptance/terraform/output.txt

```diff
@@ -1,51 +1,17 @@
 
 >>> [TERRAFORM] init -no-color -get=false
 
+Error: Failed to install provider
```

Comment:
Ignore for now. We'll either remove this test or make this easier to maintain (by disabling it). Have not looked into why this is failing.